src/main/resources/templates/db_scripts/schema.json

```diff
@@ -357,6 +357,11 @@
             "fieldName" : "acquisitionsUnitId"
          }
        ]
+     },
+     {
+      "tableName": "order_templates",
```

Comment:
Even though it is not mentioned in jira, I think it makes sense to make `templateName` unique. UI displays only names in DDL and if there are several identical names, I do not think end users would be happy... 
@craigmcnally what do you think?

Reply:
sure, makes sense to me